app/scripts/directives/podDonut.js

```diff
@@ -153,6 +154,13 @@ angular.module('openshiftConsole')
             return 'Terminating';
           }
 
+          // check for 'Failed' phase before checking if pod isTroubled
+          // as isFailingPodFilter shares some of the same container statuses
+          // as isTroubledPodFilter
+          if(isFailingPodFilter(pod)) {
```

Comment:
Though this is the easiest way to do this, I was not sure if it was better to have `isTroubledPodFilter` *not* check for the same container errors that `isFailingPodFilter` checks for, and update conditional statements in places like [pods-table.html](https://github.com/openshift/origin-web-console/blob/master/app/views/directives/pods-table.html#L18), for example, to display a `tooltip` with the container error if `isTroubledPod || isFailingPod`